apiserver/facades/agent/uniter/uniter.go

```diff
@@ -1599,6 +1616,12 @@ func (u *UniterAPI) SLALevel() (params.StringResult, error) {
 }
 
 // NetworkInfo returns network interfaces/addresses for specified bindings.
+func (u *UniterAPIV6) NetworkInfo(args params.NetworkInfoParams) (params.NetworkInfoResults, error) {
```

Comment:
The semantics *aren't* different, though, precisely because you're just calling the other one without any additional validation. As it is, you may as well not have this, and just allow the V7 method to be exposed to V6 clients. If you add `args.RelationId = nil` here, it would make it equivalent to the old behaviour.